_docs/setup/install-kubernetes.md

```diff
@@ -127,108 +127,42 @@ default namespace. They can be modified for deployment in a different namespace.
-
-After running some services -- for example, after installing the [BookInfo]({{home}}/docs/samples/bookinfo.html)  sample application and generating some load on the application (e.g., executing `curl` requests in a `while` loop) -- the resulting service graph should look something like this:
-
-<figure><img src="./img/servicegraph.png" alt="BookInfo Service Graph" title="BookInfo Service Graph" />
-<figcaption>BookInfo Service Graph</figcaption></figure>
-
-
 ## Verifying the installation
 
 1. Ensure the following Kubernetes services were deployed: "istio-pilot", "istio-mixer", "istio-ingress", "istio-egress",
-   "istio-ca" (if Istio Auth is enabled), and, optionally, "grafana", "prometheus' and "servicegraph").
+   "istio-ca" (if Istio Auth is enabled).
 
    ```bash
    kubectl get svc
    ```
    ```bash
    NAME            CLUSTER-IP      EXTERNAL-IP       PORT(S)                       AGE
-   grafana         10.83.252.16    <none>            3000:30432/TCP                5h
    istio-egress    10.83.247.89    <none>            80/TCP                        5h
    istio-ingress   10.83.245.171   35.184.245.62     80:32730/TCP,443:30574/TCP    5h
    istio-pilot     10.83.251.173   <none>            8080/TCP,8081/TCP             5h
    istio-mixer     10.83.244.253   <none>            9091/TCP,9094/TCP,42422/TCP   5h
    kubernetes      10.83.240.1     <none>            443/TCP                       36d
-   prometheus      10.83.247.221   <none>            9090:30398/TCP                5h
-   servicegraph    10.83.242.48    <none>            8088:31928/TCP                5h
    ```
 
    Note that if your cluster is running in an environment that does not support an external load balancer
    (e.g., minikube), the `EXTERNAL-IP` of `istio-ingress` will say `<pending>` and you will need to access the
    application using the service NodePort or port-forwarding instead.
 
 2. Check the corresponding Kubernetes pods were deployed and all containers are up and running:
-   "istio-pilot-\*", "istio-mixer-\*", "istio-ingress-\*", "istio-egress-\*", "istio-ca-\*" (if Istio Auth is enabled),
-   and, optionally, "grafana-\*", "prometheus-\*' and "servicegraph-\*".
+   "istio-pilot-\*", "istio-mixer-\*", "istio-ingress-\*", "istio-egress-\*", "istio-ca-\*" (if Istio Auth is enabled).
 
    ```bash
    kubectl get pods
    ```
    ```bash
-   grafana-3836448452-vhc1v         1/1       Running   0          5h
    istio-ca-3657790228-j21b9        1/1       Running   0          5h
    istio-egress-1684034556-fhw89    1/1       Running   0          5h
    istio-ingress-1842462111-j3vcs   1/1       Running   0          5h
    istio-pilot-2275554717-93c43     2/2       Running   0          5h
    istio-mixer-2104784889-20rm8     1/1       Running   0          5h
-   prometheus-3067433533-wlmt2      1/1       Running   0          5h
-   servicegraph-3127588006-pc5z3    1/1       Running   0          5h
    ```
 
 
-## Deploy your application
```

Comment:
same for this, should at least be a link ?

Reply:
This is a setup tab. There is a dedicated install sidecar section. We will add next steps once this section is done. Cannot finish everything in one doc.

Reply:
I'm not a fan of disappearing content before the replacement is made